cmd/kueueviz/frontend/src/useWebSocket.js

```diff
@@ -38,7 +38,13 @@ const useWebSocket = (url) => {
 
     ws.onerror = (err) => {
       console.error("WebSocket error:", err);
-      setError("WebSocket connection error");
+      const errorMessage = `
+      Failed to fetch data from WebSocket: ${err.message}.
```

Comment:
```suggestion
      Failed to fetch data from WebSocket: ${err.message}.
```
On the screenshot, err.message is undefined. Maybe we should add this message only if err.message exists